terraform/modules/tf_stream_alert_cloudtrail/main.tf

```diff
@@ -1,3 +1,87 @@
+// KMS key for encrypting CloudTrail logs
+resource "aws_kms_key" "cloudtrail_encryption" {
+  description         = "Encrypt Cloudtrail logs for ${var.prefix}.${var.cluster}.streamalert.cloudtrail"
+  policy              = "${data.aws_iam_policy_document.cloudtrail_encryption.json}"
+  enable_key_rotation = true
+}
+
+data "aws_iam_policy_document" "cloudtrail_encryption" {
+  statement {
+    sid = "Enable IAM User Permissions"
```

Comment:
Isn't this policy to allow root to manage the key?

Reply:
Sort of - it's a required policy that enables any IAM user in the account (with the appropriate permissions) to manage the key - https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html#key-policy-default-allow-root-enable-iam